geometry_msgs/Pose tcp_goal
---
uint8[] return # Pickled return